### DISCUSSION TITLE
At the mention of the 'gender' variable, the copilot completely refuses to show additions.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#55478] [Copilot Conversations]

**Comment by yourcasualdev** [comment]
It has happened to me too. In my java homework. Copilot refuses to give anything about gender related definitions, functions anything.

**Comment by mohdrasbi** [comment]
Same thing just happened to me.


https://github.com/community/community/assets/24537202/3dcb16ed-725b-4fe3-a4af-6a5907df827e

**Comment by monsterfurby** [comment]
I have the same issue. Sure, there are workarounds (renaming the variable), but considering that this is a very common variable in real-world, perfectly ordinary workflows, I'm pretty sure this is not how it should work.

Since changing the term to "sex" yields the same result, I would speculate that perhaps the content filter incorrectly sees both terms as synonyms and strips the term from the prompt it passes to the LLM.

**Comment by dansantner** [comment]
I have a bunch of variables that copilot happily generated everything for my react form properly but when it came to the gender field....crickets!
